cloudbuild/scorecard-tag.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Security Scorecard Authors
```

Comment:
Curious, how does this file get used?

Reply:
Scorecard docker images needs to be built for git tags.

Reply:
Scorecard-tag is used whenever there is a tag pushed into main. docker pull gcr.io/openssf/scorecard:v1.0.0